src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/Microsoft.VisualStudio.Mac.RazorAddin.csproj

```diff
@@ -82,11 +82,6 @@
     <MPackFile Include="$(ArtifactsBinDir)Microsoft.VisualStudio.Mac.RazorAddin\$(Configuration)\net472\Microsoft.Extensions.Configuration.dll" />
     <MPackFile Include="$(ArtifactsBinDir)Microsoft.VisualStudio.Mac.RazorAddin\$(Configuration)\net472\Microsoft.Extensions.Configuration.Abstractions.dll" />
     <MPackFile Include="$(ArtifactsBinDir)Microsoft.VisualStudio.Mac.RazorAddin\$(Configuration)\net472\Microsoft.Extensions.Logging.dll" />
-    <MPackFile Include="$(ArtifactsBinDir)Microsoft.VisualStudio.Mac.RazorAddin\$(Configuration)\net472\Microsoft.Extensions.Logging.Abstractions.dll" />
```

Comment:
I'm removing this from the mpack file BUT we're not removing it from our addin XML files:
- [addin.info](https://github.com/dotnet/razor-tooling/blob/1b38348c65e67a844e6691a26cabc95eac622036/src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/mpack/addin.info)
- [addin manifest](https://github.com/dotnet/razor-tooling/blob/main/src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/Properties/_Manifest.addin.xml)
    - I don't actually know the difference between this and addin.info maybe @KirillOsenkov knows?

Reply:
I think addin.info isn't used by much and could be redundant (?) I think it was only used by the extension gallery and since we don't publish to the gallery I guess we don't care?

Reply:
@Therzok for a sanity check. I think it's safe to remove these .dlls from the .mpack as the IDE will load them from build/bin, right? Also need to leave the Runtime entries in the _Manifest.addin.xml as we want to indicate to the runtime to load them, right?

Reply:
@KirillOsenkov I think the design requires your addin to depend on the actual loader.

Reply:
So, `Runtime Import` if you ship, `Addin Dependency` if some other addin ships and loads.

Reply:
> So, `Runtime Import` if you ship, `Addin Dependency` if some other addin ships and loads.

Hmmm, if the dll becomes part of the "core" set of dlls [as referenced here](https://github.com/xamarin/vsmac/pull/6484#issuecomment-1087788031) do we still use `Runtime Import`?

Reply:
ah interesting, thanks for clarifying!